hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/MetricUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.metrics;
+
+import org.apache.hudi.common.util.collection.Pair;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MetricUtils {
+
+  private static Pair<String, String> splitToPair(String label) {
+    String[] keyValuess = label.split(":");
```

Comment:
split can return empty string.. does it make sense to have empty key and value in that case?

Reply:
The caller of `splitToPair` is filtering strings which have `:` before passing to function. But in a weird case where input string is `:`, key and value can both be empty. Adding a validation check for this.